README.md

```diff
@@ -26,7 +26,22 @@
           "app_category_type": "public.app-category.productivity"
         },
         "linux": {
-          "icon": "resources/linux/<your-project-id>.png"
+          "icon": "resources/linux/<your-project-id>.png",
+          "debian_config": {
+            "section": "Databases",
+            "depends": [
```

Comment:
Scratch those thoughts for now at least, PEBCAK.

Will look more closely and see if we don't need COMPASS-590 at all which would be nice.